src/v/cluster/members_backend.cc

```diff
@@ -177,8 +177,10 @@ ss::future<> members_backend::reconciliation_loop() {
               "{}",
               std::current_exception());
         }
-        // when an error occurred wait before next retry
-        co_await ss::sleep_abortable(_retry_timeout, _as.local());
+        if (!_as.local().abort_requested()) {
+            // when an error occurred wait before next retry
+            co_await ss::sleep_abortable(_retry_timeout, _as.local());
+        }
```

Comment:
what's the goal with this change here? if `as` hasn't been aborted, then this won't prevent `sleep_aborted` from being thrown later. if `as` were aborted  and we called sleep, then would we just get `sleep_aborted` immediately without sleeping?

Reply:
yeah. this would have been better wrapped in a `try`/`catch` block as well